api/app/src/mappings/dexcom/biometrics.ts

```diff
@@ -0,0 +1,52 @@
+import { Biometrics } from "@metriport/api";
+import { Sample } from "@metriport/api/lib/devices/models/common/sample";
+
+import { DexcomEvgs } from "./models/evgs";
+import { PROVIDER_DEXCOM } from "../../shared/constants";
+import { Util } from "../../shared/util";
+
+export const mapToBiometrics = (dexcomBiometrics: DexcomEvgs, date: string): Biometrics => {
+  const metadata = {
+    date: date,
+    source: PROVIDER_DEXCOM,
+  };
+
+  const biometrics: Biometrics = {
+    metadata: metadata,
+  };
+
+  const samples = dexcomBiometrics.records.reduce((acc: Sample[], curr) => {
+    if (curr.value) {
+      acc.push({
+        time: curr.displayTime,
+        value: curr.value,
+      });
+    }
+
+    return acc;
+  }, []);
+
+  const mgDlValues = dexcomBiometrics.records.reduce((acc: number[], curr) => {
+    const unitMgDl = "mg/dL";
+    const unitMmolL = "mmol/L";
+    const conversion = 18.0182;
```

Comment:
I'm assuming `convert-units` doesn't do this one, right?

Reply:
Yea it doesnt have this conversion